lib/trader/trade-utils.ts

```diff
@@ -38,7 +38,7 @@ export function getFutbinSellPrice(price: futbin.Price): SellPrice {
   }
 
   // ignore old prices
-  if (price.updatedMinsAgo === -1 || price.updatedMinsAgo > 12 * 60) {
+  if (price.updatedMinsAgo === -1 || price.updatedMinsAgo > 60) {

```

Comment:
I'm also thinking to investigate futbin prices. On SBC release dates, it's just garbage. No check we can make justifies futbin price on those days. So I'm thinking maybe we can check active SBCs and if they are up less than 24 hours, ignore futbin 100%. What do you think?